packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3641,7 +3644,7 @@ export class ContainerRuntime
 				break;
 			case ContainerMessageType.IdAllocation:
 				// Remove the stashedState from the op if it's a stashed op
-				if (contents.stashedState !== undefined) {
+				if ("stashedState" in contents) {
```

Comment:
@justus-camp FYI

Reply:
The reason is `in` plays nicer with types (matters for upcoming PR), and also covers the (theoretical) case where someone did `contents.stashedState === undefined`.  That would be preserved in the old code but rightly deleted in the new code. Hopefully the difference doesn't matter in practice, but still nice to be stricter.